src/main/resources/localizations/fr.yml

```diff
@@ -4,61 +4,61 @@ mcb:
     invalid-arguments: "Arguments invalides."
     invalid-tc: "Salon textuel invalide pour la commande."
     no-alias: "Il n'y a pas d'autre alias pour cette commande."
-    fatal-error: "A fatal error have occurred before %s command execution finishes."
+    fatal-error: "Une erreur fatale a été détectée avant que l'éxécution de la commande %s soit finie."
   commands:
     perm:
-      description: "Manage permissions for categories."
-      invalid-sub: "% is not a sub command"
-      no-role: "%s role does not exist in this guild."
+      description: "Gérer permissions pour les catégories."
+      invalid-sub: "% n'est une sous commande"
+      no-role: "Le role %s n'existe pas dans cette guilde."
       sub:
         set:
-          description: "Set a role for certain categories. \nUsage: {+} to allow {-} to prevent. \nFlags: {-<command|category|all>} or {+<command|category|all}. \nie: {-tps} | {+info}"
-          invalid-flag: "%s is an invalid flag."
-          flag-not-null: "You must provide a flag(s)!"
-          all-invalid-flags: "All provided flags are invalid."
-          success: "Successfully set permissions for %s"
-          omitted-flags: "The following flag(s) is/are invalid and have been omitted: %s"
-          added: "The following have been added: %s"
+          description: "Mettre un rôle pour certaines catégories. \nUsage: {+} pour autoriser {-} pour interdire. \nFlags: {-<command|category|all>} or {+<command|category|all}. \nie: {-tps} | {+info}"
+          invalid-flag: "%s est un flag invalide."
+          flag-not-null: "Tu dois mettre un ou plusieurs flag(s)!"
+          all-invalid-flags: "Tous les flags indiqués sont invalides."
+          success: "Permission(s) mise(s) avec succès pour le role %s"
+          omitted-flags: "Le(s) flag(s) suivant(s) est/sont invalide(s) et ont été ommis: %s"
+          added: "Les flags suivants ont été ajoutés: %s"
         check:
-          description: "Check the current permission flags of a role."
-          all-description: "All roles that have permission flags and their corresponding flags"
+          description: "Regarder les permissions actuelles pour un rôle."
+          all-description: "Tous les rôles qui ont des permissions et leurs flags."
           success: "%s permission flags."
```

Comment:
Untranslated? May not need to be though...

Reply:
%s is the role in this case. So maybe translate it to `indicateur d'autorisation du %s`. I'm not good with french :3.